packages/block-library/src/rss/style.scss

```diff
@@ -1,4 +1,7 @@
-.wp-block-rss {
+ul.wp-block-rss { // The ul is needed for specificity to override the reset styles in the editor.
```

Comment:
We don't need this if https://github.com/WordPress/gutenberg/pull/32659 lands (which I really hope it does)

Reply:
#32659 needs to land, I agree. In reviews an issue with the gallery block has been surfaced, that I want to poke at in more depth before I merge, and since I'm going on vacation today, I've held off on merging it even if I have two thumbs up to do so, as I want to be able to follow up.

I expect to do that followup when I get back from AFK. But if there's great urgency about landing it, just important to note there's [a gallery aspect that needs followup](https://github.com/WordPress/gutenberg/pull/32659#issuecomment-876357415).